pkg/virt-controller/watch/pool/pool.go

```diff
@@ -515,6 +516,43 @@ func filterVMs(vms []*virtv1.VirtualMachine, f func(vmi *virtv1.VirtualMachine)
 	return filtered
 }
 
+func resolveDownscaleStrategy(policy *poolv1.VirtualMachinePoolDownscalePolicy) poolv1.VirtualMachinePoolDownscaleStrategy {
+	if policy == nil || policy.Strategy == "" {
+		return poolv1.VirtualMachinePoolDownscaleStrategyRandom
+	}
+	return policy.Strategy
+}
+
+func sortVMsForDownscale(vms []*virtv1.VirtualMachine, strategy poolv1.VirtualMachinePoolDownscaleStrategy) {
+	switch strategy {
+	case poolv1.VirtualMachinePoolDownscaleStrategyOrdered:
```

Comment:
The word `Ordered` very context dependent.  I think we need to be more specific.  Something like `DescendingOrder` sounds better to me